workflow/tests/resnet_hers_test.rb

```diff
@@ -44,8 +44,10 @@ def test_resnet_ashrae_140
 
     htg_loads, clg_loads = _write_ashrae_140_results(all_results, test_results_csv)
 
-    # Check results
-    _check_ashrae_140_results(htg_loads, clg_loads)
+    # Check results if we have them all
+    if all_results.size == 26
```

Comment:
Thanks for the contribution, @rh0dium. I would prefer to use `> 1`, because if we ever added another test, it would be easy to forget to bump this number and not realize the checks aren't actually running. Would that work for you?

Reply:
You bet - And I approached it the other way. But I see your point and will update it.